packages/auto-approve/src/auto-approve.ts

```diff
@@ -162,7 +163,7 @@ export function handler(app: Probot) {
       const prConfig = await getBlobFromPRFiles(
         context.octokit,
         owner,
```

Comment:
If this is coming from a fork, this should also probably be the head branch's repo's owner as well.